src/node.cc

```diff
@@ -1061,19 +1061,15 @@ int Start(int argc, char** argv) {
     Isolate::CreateParams params;
     const std::vector<size_t>* indexes = nullptr;
     std::vector<intptr_t> external_references;
```

Comment:
I think `external_references` can be removed as it does not look like it is used anymore.